R/bias.R

```diff
@@ -89,22 +89,16 @@ bias_sample <- function(true_values, predictions) {
 #'
 #' \deqn{
 #' B_t = (1 - 2 \cdot \max \{i | q_{t,i} \in Q_t \land q_{t,i} \leq x_t\})
-#' \mathbf{1}( x_t \leq q_{t, 0.5}) \\
+#'  1( x_t \leq q_{t, 0.5}) \\
```

Comment:
not better to have `\mathbf` to distinguish the symbol from the number 1?

Reply:
as above

Reply:
```suggestion
#'  \mathbf{1}( x_t \leq q_{t, 0.5}) \\
```